src/editor/EditorContainer.vue

```diff
@@ -26,9 +28,10 @@ const onChange = debounce((code: string) => {
       :filename="store.state.activeFile.filename"
     />
     <Message
-      v-if="props.editorComponent.editorType !== 'monaco'"
+      v-if="props.editorComponent.editorType !== 'monaco' && showMessage"
       :err="store.state.errors[0]"
     />
+    <MessageToggle v-model="showMessage" />
```

Comment:
I think if it's not monaco, we can hide the toggle.

Reply:
Done